tuxemon/mod_manager/tuxemoncontentserver.py

```diff
@@ -0,0 +1,81 @@
+"""
+Mod manager support module TuxemonContentServer
+https://github.com/vXtreniusX/TuxemonContentServer
+"""
+
+import urllib.request
+import requests
+import json
+
+def update(url):
+    """Returns the response from the server"""
+    packages = requests.get(url=url)
+    print(packages.text)
+    return packages.json()
+
+def download_package(self, name, release, repo=None, dont_extract=False, install_deps=True, installed=None):
```

Comment:
this looks like it is largely duplicated in another file.  could they be refactored to remove it?